environments/development/electronic-monitoring-data-store/load-capital-blob-storage/workflow.yml

```
@@ -0,0 +1,32 @@
+---
+dag:
+  repository: moj-analytical-services/airflow-load-em-data
+  tag: v0.1.73-dev-365fcf86d45fad30a2680b0ee83a222186ee6016
+  compute_profile: general-spot-1vcpu-4gb
+  catchup: false
+  depends_on_past: false
+  schedule: "0 * * * *"
+  start_date: "2025-04-15"
+  max_active_runs: 1
+  env_vars:
+    SUPPLIER_NAME: "capita"
+    SYSTEM_NAME: "blob_storage"
+    AWS_DEFAULT_REGION: "eu-west-2"
+    AWS_ATHENA_QUERY_EXTRACT_REGION: "eu-west-2"
+    AWS_DEFAULT_EXTRACT_REGION: "eu-west-2"
+
+notifications:
+  emails:
+    - EMDataEngineering@justice.gov.uk
+  slack_channel: electronic-monitoring-data-hub-test-notifications
+
+iam:
+  external_role: arn:aws:iam::800964199911:role/airflow-dev-load-capita-blobstorage
```

```suggestion
  external_role: arn:aws:iam::800964199911:role/airflow-dev-load-capita-blob-storage
```
with the addition of dashes this will be here now